.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+
+
+  (ﾉ◕ヮ◕)ﾉ*:・ﾟ✧
+  
+  Thank you for your interest in contributing to the k6 project!
+  
+  Before you get started, we'd kindly like to ask you to read our:
+    - Contribution guidelines at https://github.com/loadimpact/k6/blob/master/CONTRIBUTING.md
+    - Code of Conduct at https://github.com/loadimpact/k6/blob/master/CODE_OF_CONDUCT.md
+    
+  Out of respect for your time, please start a discussion regarding any bigger contributions
+  either in the #contributors channel of the k6 slack or in the Github Issue before you get
```

Comment:
How about the community forum ?

Reply:
I can add that if it makes sense for you. 😅  I assumed (maybe incorrectly) that you, as core contributors, would prefer gh issues and slack.

Reply:
for most things, github issue > community forum > slack - slack would only win over the community forum if near-real-time communication would be useful, for example to debug a tricky or urgent bug, or something like that

Reply:
Done. Reapprove and i'll merge it 👍